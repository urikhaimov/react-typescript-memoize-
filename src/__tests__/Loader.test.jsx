import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../components/Loader';


test('render Loader', () => {
  const component = renderer.create(
    <Loader/>,

  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

 
});