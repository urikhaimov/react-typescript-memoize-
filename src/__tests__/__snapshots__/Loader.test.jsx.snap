// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Loader 1`] = `
<div
  className="loader"
/>
`;
